src/binmap.cpp

```diff
@@ -155,7 +155,7 @@ class SubCommand : public Command {
   }
 };
 
-struct canonicalize_path {
+struct canonicalize {
```

Comment:
why this change?

Reply:
Ah, that's a change made in the last commit, I worked on the previous one, I can change this